src/lib/components/inventory/list_item_modal.ts

```diff
@@ -8,11 +8,14 @@ import {ClientSend} from '../../bridge/client';
 import {ListItem} from '../../bridge/handlers/list_item';
 import {FetchRecommendedPrice} from '../../bridge/handlers/fetch_recommended_price';
 import {listItemModalStyles} from './list_item_modal_styles';
-import {isLoggedIntoCSFloat} from '../../utils/auth';
 import {CSFError, CSFErrorCode} from '../../utils/errors';
-import {FetchPendingTrades} from '../../bridge/handlers/fetch_pending_trades';
 import {FetchCSFloatMe} from '../../bridge/handlers/fetch_csfloat_me';
 
+// SVG icons for percentage assessment
```

Comment:
Perhaps we can move these icons to a central `icons.ts` file either in this directory or in a more global place?